common/components/BalanceSidebar/AccountInfo.tsx

```diff
@@ -80,38 +79,40 @@ export default class AccountInfo extends React.Component<Props, State> {
                 className="AccountInfo-list-item-clickable mono wrap"
                 onClick={this.toggleShowLongBalance}
               >
-                {this.state.showLongBalance
-                  ? balance ? balance.toString() : '???'
-                  : balance ? formatNumber(balance.amount) : '???'}
+                <UnitDisplay
+                  value={balance}
+                  unit={'ether'}
+                  displayShortBalance={!showLongBalance}
+                />
               </span>
-              {` ${network.name}`}
+              {network.name}
```

Comment:
Might want to have a spacer before this so balance reads as `??? ETH` instead of `???ETH`.